inspect_builder_test.go

```diff
@@ -49,13 +50,13 @@ func testInspectBuilder(t *testing.T, when spec.G, it spec.S) {
 		}
 
 		builderImage = fakes.NewImage("some/builder", "", nil)
-		h.AssertNil(t, builderImage.SetLabel("io.buildpacks.stack.id", "test.stack.id"))
-		h.AssertNil(t, builderImage.SetLabel(
+		assert.Nil(builderImage.SetLabel("io.buildpacks.stack.id", "test.stack.id"))
+		assert.Nil(builderImage.SetLabel(
 			"io.buildpacks.stack.mixins",
 			`["mixinOne", "build:mixinTwo", "mixinThree", "build:mixinFour"]`,
 		))
-		h.AssertNil(t, builderImage.SetEnv("CNB_USER_ID", "1234"))
-		h.AssertNil(t, builderImage.SetEnv("CNB_GROUP_ID", "4321"))
+		assert.Nil(builderImage.SetEnv("CNB_USER_ID", "1234"))
+		assert.Nil(builderImage.SetEnv("CNB_GROUP_ID", "4321"))
```

Comment:
any reason not to use `assert.Succeeds`?

Reply:
Nope totally missed this.